ftp/2.0/modules/ROOT/pages/ftp-connector-upgrade-migrate.adoc

```diff
@@ -15,7 +15,8 @@ Upgrade Anypoint Connector for FTP (FTP Connector) to version 2.x.
 This release contains the following changes:
 
 * The `mule-file-commons` module dependency is removed to make FTP Connector a standalone connector. 
-* The *List* operation now returns the file path instead of the file content. 
+* The *List* operation now returns the file path instead of the file content.
+* The `Time Between Size Check` and `Time Between Size Check Unit` parameters are removed from *List* operation, as the file content is no longer read. 
```

Comment:
```suggestion
* The *List* operation no longer includes the *Time Between Size Check* and *Time Between Size Check Unit* fields because the file content is no longer read.
```